app/models/levelling_up_premium_payments/award.rb

```diff
@@ -1,42 +1,2694 @@
 module LevellingUpPremiumPayments
-  # This is concerned with the award information from the spreadsheet, which will
-  # contain the pre-computed amount, thereby we don't need to check the EIA or pupil premium
-  # status ourselves.
-  #
-  # The only reason for this class to change is regarding how the monetary amount
-  # is stored.
   class Award
-    # The spreadsheet contains the pre-computed amount as a currency string
-    # like "£2,000.00" or "No payment" and is attached to
-    # https://dfedigital.atlassian.net/browse/CAPT-253
-    #
-    # We'll need a job to perform the initial database storage of this data.
+    # The spreadsheet attached to https://dfedigital.atlassian.net/browse/CAPT-253
+    # contains the pre-computed amount as a currency stringlike "£2,000.00" or
+    # "No payment".
     def initialize(school)
       raise "nil school" if school.nil?
 
       @school = school
     end
 
     def self.max
-      @@maximum_award_amount_in_pounds
+      @maximum_award_amount_in_pounds ||= urn_to_award_amount_in_pounds.values.max
+    end
+
+    def self.urn_to_award_amount_in_pounds
```

Comment:
I think we would need to be able to retrieve this by academic year of the claim, e.g.

```suggestion
    def self.urn_to_award_amount_in_pounds(academic_year)
```